phpseclib/Math/BigInteger/Engines/PHP.php

```diff
@@ -70,6 +70,13 @@
      * @access protected
      */
     const ENGINE_DIR = 'PHP';
+    
+    /**
+     * Primes > 2 and < 1000
+     *
+     * @var array
+     */
+    protected static $primes;
```

Comment:
`$primes` is defined in PHP32.php and PHP64.php. PHP.php is abstract so it can't be instantiated anyway.

The reason I defined `$primes` in PHP32.php and PHP64.php was so that you wouldn't run into problems if you tried to instantiate both of them.

Reply:
Never mind - I see you undid that in a newer commit - that you changed, later, `self::$primes` to `static::$primes`. I'd say that'd be a good PR on it's own. Maybe try adding a unit test too.